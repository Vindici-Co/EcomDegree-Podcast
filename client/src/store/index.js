import appReducer from "./reducers";
import { configureStore, getDefaultMiddleware } from "@reduxjs/toolkit";
import logger from "redux-logger";

const store = configureStore({
  reducer: appReducer,
  middleware: [logger, ...getDefaultMiddleware()],
});

export default store