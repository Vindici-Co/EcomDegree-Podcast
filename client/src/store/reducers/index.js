import userReducer from "./userReducer.js"

export const appReducer = {
  users: userReducer,
}

export default appReducer