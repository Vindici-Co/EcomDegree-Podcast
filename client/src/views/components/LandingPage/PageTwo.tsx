import React from "react";
import styled from "styled-components";
import PageTwoLego from "./assets/PageTwoLego.png";
import { Fade } from "@material-ui/core/";

const TitleDivLeft = styled.p`
	font-weight: bold;
	font-size: 75px;
	line-height: 90px;
	letter-spacing: -2.25px;
	color: white;
	font-family: ${(props) => props.theme.fonts.main};
	margin: 0;
`;

const Orange = styled.span`
	color: ${(props) => props.theme.colors.main.orange};
`;

const LeftDesc = styled.p`
	font-family: ${(props) => props.theme.fonts.secondary};
	font-size: 16px;
	font-style: normal;
	font-weight: 400;
	line-height: 24px;
	letter-spacing: 0.5px;
	text-align: left;
`;

export const PageTwoLeft: React.FC = (): JSX.Element => {
	return (
		<>
			<Fade in={true} timeout={2000}>
				<TitleDivLeft>
					Latest Youtube
					<br /> Video
					<LeftDesc>
						A Case Study on how EDU student Sam Maxwell made $10,000 his first <br />
						weeek dropshipping. Tune in to learn you can apply Samâ€™s key points of <br />
						advice to your ecommerce business.
					</LeftDesc>
				</TitleDivLeft>
			</Fade>
		</>
	);
};

export const PageTwoRight: React.FC = (): JSX.Element => {
	return (
		<>
			<Fade in={true} timeout={2000}>
				<img
					src={PageTwoLego}
					alt="PageTwoLego"
					style={{
						margin: "0px 360px 0px 0px"
					}}
				/>
			</Fade>
		</>
	);
};
