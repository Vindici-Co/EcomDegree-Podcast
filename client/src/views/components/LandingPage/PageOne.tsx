import React, { useEffect } from "react";
import styled from "styled-components";
import { makeStyles } from "@material-ui/styles";
import { Fade, useMediaQuery } from "@material-ui/core/";
import bgPageOne from "./assets/bgPageOne.png";
import { IconButton } from "@material-ui/core";
import { ArrowRight, ArrowLeft } from "@material-ui/icons";
import { IPageProps, customUseStyles } from "./LandingPage";

const PageOne: React.FC<IPageProps> = (props: IPageProps): JSX.Element => {
	const smallSize = useMediaQuery("(max-width:1200px)");
	const styleProps = { smallSize };
	const classes = customUseStyles(styleProps);

	useEffect(() => {
		console.log(smallSize);
	}, [smallSize]);

	const TitleDivLeft = styled.p`
		font-weight: bold;
		line-height: ${smallSize ? "65px" : "90px"};
		letter-spacing: -2.25px;
		color: white;
		font-family: ${(props) => props.theme.fonts.main};
		margin: 0;
	`;

	const Orange = styled.span`
		color: ${(props) => props.theme.colors.main.orange};
	`;

	const LeftDesc = styled.p`
		font-size: 16px;
		font-style: normal;
		font-weight: 400;
		line-height: 24px;
		letter-spacing: 0.5px;
		text-align: left;
		font-family: ${(props) => props.theme.fonts.secondary};
	`;

	return (
		<div
			className={classes.Container}
			style={{
				backgroundImage: `url(${bgPageOne})`,
				backgroundRepeat: "no-repeat",
				backgroundSize: "cover"
			}}
			data-testid="pageOneContainer"
		>
			<Fade in={true} timeout={1000}>
				<div className={classes.OuterLeftContainer}>
					<div className={classes.InnerLeftContainer}>
						<TitleDivLeft style={{ fontSize: smallSize ? "50px" : "75px" }}>
							<Orange>6+</Orange> Hours of <br />
							Free Game.
							<LeftDesc>
								When our team provides design and digital marketing fashion design. The
								{smallSize ? null : <br />} app provides design and digital graphic
								design.
							</LeftDesc>
						</TitleDivLeft>
					</div>
					<div className={classes.Btns}>
						<IconButton
							className={classes.minusBtn}
							onClick={() => props.pageChange(-1)}
							data-testid="minusButton"
						>
							<ArrowLeft style={{ fontSize: 50 }} />
						</IconButton>
						<IconButton
							className={classes.plusBtn}
							onClick={() => props.pageChange(1)}
							data-testid="plusButton"
						>
							<ArrowRight style={{ fontSize: 50 }} />
						</IconButton>
					</div>
				</div>
			</Fade>
			{smallSize ? null : (
				<div className={classes.RightContainer}>
					<></>
				</div>
			)}
		</div>
	);
};

export default PageOne;
