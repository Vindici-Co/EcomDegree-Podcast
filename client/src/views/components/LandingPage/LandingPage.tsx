import React, { useState } from "react";
import Navbar from "../Navbar/Navbar";
import PageOne from "./PageOne";
import PageTwo from "./PageTwo";
import PageThree from "./PageThree";
import { makeStyles } from "@material-ui/styles";

interface styleProps {
	smallSize: boolean;
}

export const customUseStyles = makeStyles({
	Container: (styleProps: styleProps) => ({
		height: "100vh",
		width: "100vw",
		display: "flex",
		flexDirection: styleProps.smallSize ? "column" : "row",
		justifyContent: styleProps.smallSize ? "center" : "space-around",
		alignItems: "center",
		padding: "0",
		margin: "0",
		color: "white",
		// overflow: styleProps.smallSize ? "hidden" : ""
		top: 0
	}),

	OuterLeftContainer: {
		display: "flex",
		flexDirection: "column"
	},

	InnerLeftContainer: (styleProps: styleProps) => ({
		display: "flex",
		flexDirection: "column",
		justifyContent: "center",
		width: styleProps.smallSize ? "100%" : "743.688px",
		height: styleProps.smallSize ? "100%" : "349.969px",
		margin: "0"
	}),
	Btns: {
		display: "flex",
		width: "auto",
		color: "white"
	},
	RightContainer: (styleProps: styleProps) => ({
		width: "216.41px",
		display: styleProps.smallSize ? "none" : "flex",
		flexDirection: "column",
		justifyContent: "center",
		alignItems: "center",
		height: "96%"
	}),
	minusBtn: {
		background: "#F49A12",
		color: "white",
		height: "56px",
		width: "60px",
		left: " 0px",
		top: " 0px",
		borderRadius: "20px",
		marginRight: "20px",
		"&:hover": {
			background: "#e3a880"
		}
	},
	plusBtn: {
		background: "#F49A12",
		color: "white",
		height: "56px",
		width: "60px",
		left: " 0px",
		top: " 0px",
		borderRadius: "20px",
		"&:hover": {
			background: "#e3a880"
		}
	}
});

export interface IPageProps {
	pageChange(event: number): void;
}

const LandingPage: React.FC = () => {
	const [page, setPage] = useState(0);

	const pageChange = (event: number): void => {
		const NUM_PAGES = 3;
		let newPage = (page + event) % NUM_PAGES;
		while (newPage < 0) {
			newPage += NUM_PAGES;
		}
		setPage(newPage);
	};

	const newPages = [
		<PageOne key={0} pageChange={pageChange} />,
		<PageTwo key={1} pageChange={pageChange} />,
		<PageThree key={2} pageChange={pageChange} />
	];

	return (
		<div data-testid="landingPageContainer">
			<Navbar />
			{newPages[page]}
		</div>
	);
};
export default LandingPage;
