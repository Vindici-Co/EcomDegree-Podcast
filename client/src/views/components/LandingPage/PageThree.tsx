import React from "react";
import styled from "styled-components";
import PageThreeThumbnail from "./assets/PageThreeThumbnail.png";
import { ReactComponent as ApplePodcasts } from "./assets/ApplePodcastsSvg.svg";
import { IconButton } from "@material-ui/core";
import { ArrowRight, ArrowLeft } from "@material-ui/icons";
import { makeStyles, useTheme } from "@material-ui/styles";
import { Fade, useMediaQuery } from "@material-ui/core/";
import { IPageProps, customUseStyles } from "./LandingPage";

const PageThree: React.FC<IPageProps> = (props: IPageProps): JSX.Element => {
	const smallSize = useMediaQuery("(max-width:1200px)");
	const styleProps = { smallSize };
	const classes = customUseStyles(styleProps);

	const TitleDivLeft = styled.p`
		font-family: ${(props) => props.theme.fonts.main};
		font-size: ${smallSize ? "50px" : "75px"};
		font-style: normal;
		font-weight: 700;
		line-height: ${smallSize ? "65px" : "90px"};
		letter-spacing: -2.25px;
		text-align: left;
		margin: 0;
	`;

	const Orange = styled.span`
		color: ${(props) => props.theme.colors.main.orange};
	`;

	const LeftDesc = styled.p`
		font-family: ${(props) => props.theme.fonts.secondary};
		font-size: 16px;
		font-style: normal;
		font-weight: 400;
		line-height: 24px;
		letter-spacing: 0.5px;
		text-align: left;
	`;

	return (
		<>
			<div
				className={classes.Container}
				style={{
					backgroundColor: "#030314",
					backgroundRepeat: "no-repeat",
					backgroundSize: "cover"
				}}
				data-testid="pageThreeContainer"
			>
				<Fade in={true} timeout={1000}>
					<div className={classes.OuterLeftContainer}>
						<div className={classes.InnerLeftContainer}>
							{smallSize ? null : (
								<ApplePodcasts
									style={{
										height: "58.18px",
										width: "240px"
									}}
								/>
							)}

							<TitleDivLeft>
								Listen Now <br />
								<Orange>Mindset Mastery</Orange>.
								{smallSize ? (
									<ApplePodcasts
										style={{
											width: "183px",
											height: "44.36px",
											marginTop: "15px"
										}}
									/>
								) : null}
								<LeftDesc>
									When our team provides design and digital marketing fashion
									{smallSize ? null : <br />}design. The app provides design and digital
									graphic design.
								</LeftDesc>
							</TitleDivLeft>
						</div>
						<div className={classes.Btns}>
							<IconButton
								className={classes.minusBtn}
								onClick={() => props.pageChange(-1)}
								data-testid="minusButton"
							>
								<ArrowLeft style={{ fontSize: 50 }} />
							</IconButton>
							<IconButton
								className={classes.plusBtn}
								onClick={() => props.pageChange(1)}
								data-testid="plusButton"
							>
								<ArrowRight style={{ fontSize: 50 }} />
							</IconButton>
						</div>
					</div>
				</Fade>
				<Fade in={true} timeout={1000}>
					<div className={classes.RightContainer}>
						<img
							src={PageThreeThumbnail}
							alt="PageThreeThumbnail"
							style={{
								margin: "0px 260px 0px 0px",
								overflow: "hidden"
							}}
						/>
					</div>
				</Fade>
			</div>
		</>
	);
};

export default PageThree;
