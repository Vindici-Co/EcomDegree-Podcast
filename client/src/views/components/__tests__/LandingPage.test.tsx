import React from "react";
import { render, cleanup, fireEvent } from "@testing-library/react";
import LandingPage from "../LandingPage/LandingPage";
import store from "../../../store";
import { Provider } from "react-redux";
import { ThemeProvider } from "styled-components";
import theme from "../../../styles/theme";
import PageOne from "../LandingPage/PageOne";
import PageTwo from "../LandingPage/PageTwo";
import PageThree from "../LandingPage/PageThree";

afterEach(cleanup);

const mockPageChange = jest.fn((event: number) => event + 1);

const pageChange = (event: number, prevPage: number) => {
	const NUM_PAGES = 3;

	let newPage = (prevPage + event) % NUM_PAGES;
	while (newPage < 0) {
		newPage += NUM_PAGES;
	}
	return newPage;
};

test("<LandingPage/> renders without errors", () => {
	const { getByTestId, container } = render(
		<ThemeProvider theme={theme}>
			<Provider store={store}>
				<LandingPage />
			</Provider>
		</ThemeProvider>
	);

	expect(getByTestId("landingPageContainer")).toBeTruthy();
	expect(container).toMatchSnapshot();

	let page = 0;
	page = pageChange(1, page);
	expect(page).toBe(1);

	page = pageChange(1, page);
	expect(page).toBe(2);

	page = pageChange(1, page);
	expect(page).toBe(0);

	page = 0;
	page = pageChange(-1, page);
	expect(page).toBe(2);

	page = pageChange(-1, page);
	expect(page).toBe(1);

	page = pageChange(-1, page);
	expect(page).toBe(0);
});

test("LandingPage component: <PageOne/> renders without errors", () => {
	const { getByTestId, container } = render(
		<ThemeProvider theme={theme}>
			<Provider store={store}>
				<PageOne pageChange={mockPageChange} />
			</Provider>
		</ThemeProvider>
	);

	expect(getByTestId("pageOneContainer")).toBeTruthy();
	expect(container).toMatchSnapshot();

	const plusButton = getByTestId("plusButton");
	const minusButton = getByTestId("minusButton");
	expect(plusButton).toBeTruthy();
	expect(minusButton).toBeTruthy();

	fireEvent.click(plusButton);
	expect(mockPageChange).toHaveBeenCalledWith(1);

	fireEvent.click(minusButton);
	expect(mockPageChange).toHaveBeenCalledWith(-1);
});

test("LandingPage component: <PageTwo/> renders without errors", () => {
	const { getByTestId, container } = render(
		<ThemeProvider theme={theme}>
			<Provider store={store}>
				<PageTwo pageChange={mockPageChange} />
			</Provider>
		</ThemeProvider>
	);

	expect(getByTestId("pageTwoContainer")).toBeTruthy();
	expect(container).toMatchSnapshot();

	const plusButton = getByTestId("plusButton");
	const minusButton = getByTestId("minusButton");
	expect(plusButton).toBeTruthy();
	expect(minusButton).toBeTruthy();

	fireEvent.click(plusButton);
	expect(mockPageChange).toHaveBeenCalledWith(1);

	fireEvent.click(minusButton);
	expect(mockPageChange).toHaveBeenCalledWith(-1);
});

test("LandingPage component: <PageThree/> renders without errors", () => {
	const { getByTestId, container } = render(
		<ThemeProvider theme={theme}>
			<Provider store={store}>
				<PageThree pageChange={mockPageChange} />
			</Provider>
		</ThemeProvider>
	);

	expect(getByTestId("pageThreeContainer")).toBeTruthy();
	expect(container).toMatchSnapshot();

	const plusButton = getByTestId("plusButton");
	const minusButton = getByTestId("minusButton");
	expect(plusButton).toBeTruthy();
	expect(minusButton).toBeTruthy();

	fireEvent.click(plusButton);
	expect(mockPageChange).toHaveBeenCalledWith(1);

	fireEvent.click(minusButton);
	expect(mockPageChange).toHaveBeenCalledWith(-1);
});
