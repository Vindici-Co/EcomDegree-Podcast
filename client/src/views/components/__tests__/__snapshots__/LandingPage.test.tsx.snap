// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage/> renders without errors 1`] = `
<div>
  <div
    data-testid="landingPageContainer"
  >
    <div
      data-testid="navbarContainer"
    >
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionAbsolute MuiAppBar-colorPrimary makeStyles-AppBar-1 MuiPaper-elevation0"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        >
          <svg>
            NavLogo.svg
          </svg>
          <div
            style="margin: 0px 30px 0px 30px;"
          >
            <svg>
              Pipe.svg
            </svg>
          </div>
          <div
            class="MuiGrid-root makeStyles-tabsGrid-7 MuiGrid-container"
          >
            <div
              class="MuiTabs-root makeStyles-tabs-2"
            >
              <div
                class="MuiTabs-scroller MuiTabs-fixed"
                style="overflow: hidden;"
              >
                <div
                  class="MuiTabs-flexContainer"
                  role="tablist"
                >
                  <button
                    aria-selected="true"
                    class="MuiButtonBase-root MuiTab-root makeStyles-tabOverrides-5 MuiTab-textColorInherit makeStyles-activeTab-4 Mui-selected"
                    role="tab"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      Home
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    aria-selected="false"
                    class="MuiButtonBase-root MuiTab-root makeStyles-tabOverrides-5 MuiTab-textColorInherit makeStyles-tab-3"
                    role="tab"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      Learn
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    aria-selected="false"
                    class="MuiButtonBase-root MuiTab-root makeStyles-tabOverrides-5 MuiTab-textColorInherit makeStyles-tab-3"
                    role="tab"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      Courses
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    aria-selected="false"
                    class="MuiButtonBase-root MuiTab-root makeStyles-tabOverrides-5 MuiTab-textColorInherit makeStyles-tab-3"
                    role="tab"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      FAQ
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    aria-selected="false"
                    class="MuiButtonBase-root MuiTab-root makeStyles-tabOverrides-5 MuiTab-textColorInherit makeStyles-tab-3"
                    role="tab"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      Contact
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </div>
                <span
                  class="PrivateTabIndicator-root-18 PrivateTabIndicator-colorSecondary-20 MuiTabs-indicator makeStyles-indictator-10"
                  style="left: 0px; width: 0px;"
                />
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root makeStyles-iconGrid-6 MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root makeStyles-icon-8 MuiIconButton-edgeEnd"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiIconButton-root makeStyles-icon-8 MuiIconButton-edgeEnd"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <div
              style="margin: 0px 30px 0px 30px;"
            >
              <svg>
                Pipe.svg
              </svg>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-navBtn-9"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                Access Course
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </header>
      <div
        class="PrivateSwipeArea-root-32 PrivateSwipeArea-anchorBottom-36"
        style="height: 20px;"
      />
    </div>
    <div
      class="makeStyles-Container-22 makeStyles-Container-29"
      data-testid="pageOneContainer"
      style="background-image: url(bgPageOne.png); background-repeat: no-repeat; background-size: cover;"
    >
      <div
        class="makeStyles-OuterLeftContainer-23"
        style="opacity: 1; webkit-transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      >
        <div
          class="makeStyles-InnerLeftContainer-24 makeStyles-InnerLeftContainer-30"
        >
          <p
            class="sc-bdfBwQ jfYkcJ"
            style="font-size: 75px;"
          >
            6+ Hours of 
            <br />
            <span
              class="sc-gsTCUz FJRCo"
            >
              Free Game.
            </span>
            <p
              class="sc-dlfnbm eOPBKP"
            >
              When our team provides design and digital marketing fashion design. The
              <br />
              app provides design and digital graphic design.
            </p>
          </p>
        </div>
        <div
          class="makeStyles-Btns-25"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root makeStyles-minusBtn-27"
            data-testid="minusButton"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                style="font-size: 50px;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M14 7l-5 5 5 5V7z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButtonBase-root MuiIconButton-root makeStyles-plusBtn-28"
            data-testid="plusButton"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                style="font-size: 50px;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 17l5-5-5-5v10z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
      <div
        class="makeStyles-RightContainer-26 makeStyles-RightContainer-31"
      />
    </div>
  </div>
</div>
`;

exports[`LandingPage component: <PageOne/> renders without errors 1`] = `
<div>
  <div
    class="makeStyles-Container-37 makeStyles-Container-44"
    data-testid="pageOneContainer"
    style="background-image: url(bgPageOne.png); background-repeat: no-repeat; background-size: cover;"
  >
    <div
      class="makeStyles-OuterLeftContainer-38"
      style="opacity: 1; webkit-transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="makeStyles-InnerLeftContainer-39 makeStyles-InnerLeftContainer-45"
      >
        <p
          class="sc-bdfBwQ jfYkcJ"
          style="font-size: 75px;"
        >
          6+ Hours of 
          <br />
          <span
            class="sc-gsTCUz FJRCo"
          >
            Free Game.
          </span>
          <p
            class="sc-dlfnbm eOPBKP"
          >
            When our team provides design and digital marketing fashion design. The
            <br />
            app provides design and digital graphic design.
          </p>
        </p>
      </div>
      <div
        class="makeStyles-Btns-40"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root makeStyles-minusBtn-42"
          data-testid="minusButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              style="font-size: 50px;"
              viewBox="0 0 24 24"
            >
              <path
                d="M14 7l-5 5 5 5V7z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiIconButton-root makeStyles-plusBtn-43"
          data-testid="plusButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              style="font-size: 50px;"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
    <div
      class="makeStyles-RightContainer-41 makeStyles-RightContainer-46"
    />
  </div>
</div>
`;

exports[`LandingPage component: <PageThree/> renders without errors 1`] = `
<div>
  <div
    class="makeStyles-Container-57 makeStyles-Container-64"
    data-testid="pageThreeContainer"
    style="background-color: rgb(3, 3, 20); background-repeat: no-repeat; background-size: cover;"
  >
    <div
      class="makeStyles-OuterLeftContainer-58"
      style="opacity: 1; webkit-transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="makeStyles-InnerLeftContainer-59 makeStyles-InnerLeftContainer-65"
      >
        <img
          alt="ApplePodcasts"
          src="ApplePodcasts.png"
          style="height: 58.18px; width: 240px;"
        />
        <p
          class="sc-gKsewC jokwbq"
          style="font-size: 75px;"
        >
          Listen Now 
          <br />
          <span
            class="sc-iBPRYJ ddIViO"
          >
            Mindset Mastery
          </span>
          <p
            class="sc-fubCfw bYKeHr"
          >
            When our team provides design and digital marketing fashion
            <br />
             design. The app provides design and digital graphic design.
          </p>
        </p>
      </div>
      <div
        class="makeStyles-Btns-60"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root makeStyles-minusBtn-62"
          data-testid="minusButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              style="font-size: 50px;"
              viewBox="0 0 24 24"
            >
              <path
                d="M14 7l-5 5 5 5V7z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiIconButton-root makeStyles-plusBtn-63"
          data-testid="plusButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              style="font-size: 50px;"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
    <div
      class="makeStyles-RightContainer-61 makeStyles-RightContainer-66"
      style="opacity: 1; webkit-transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <img
        alt="PageThreeThumbnail"
        src="PageThreeThumbnail.png"
        style="margin: 0px 260px 0px 0px; overflow: hidden;"
      />
    </div>
  </div>
</div>
`;

exports[`LandingPage component: <PageTwo/> renders without errors 1`] = `
<div>
  <div
    class="makeStyles-Container-47 makeStyles-Container-54"
    data-testid="pageTwoContainer"
    style="background-image: url(bgPageTwo.png); background-repeat: no-repeat; background-size: cover;"
  >
    <div
      class="makeStyles-OuterLeftContainer-48"
      style="opacity: 1; webkit-transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="makeStyles-InnerLeftContainer-49 makeStyles-InnerLeftContainer-55"
      >
        <p
          class="sc-hKgILt gqkqcH"
        >
          Latest Youtube
          <br />
           Video
          <p
            class="sc-jSgupP kAFvVX"
          >
            A Case Study on how EDU student Sam Maxwell made $10,000 his first 
            <br />
            weeek dropshipping. Tune in to learn you can apply Sam’s key points of
             
            <br />
            advice to your ecommerce business.
          </p>
        </p>
      </div>
      <div
        class="makeStyles-Btns-50"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root makeStyles-minusBtn-52"
          data-testid="minusButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              style="font-size: 50px;"
              viewBox="0 0 24 24"
            >
              <path
                d="M14 7l-5 5 5 5V7z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiIconButton-root makeStyles-plusBtn-53"
          data-testid="plusButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              style="font-size: 50px;"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
    <div
      class="makeStyles-RightContainer-51 makeStyles-RightContainer-56"
      style="opacity: 1; webkit-transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <img
        alt="PageTwoLego"
        src="PageTwoLego.png"
        style="margin: 0px 360px 0px 0px;"
      />
    </div>
  </div>
</div>
`;
